### ISSUE TITLE
Update ga4gh/server to support GenotypePhenotype schema

### ISSUE THREAD

**Issue by bwalsh** [ga4gh/g2p-team#3]
While ga4gh/schema was [updated](https://github.com/ga4gh/schemas/commit/e87bfc256ab54bc6ee3ebccb0d82a5b145aa1889#diff-5ffc45f746567e5fb402c64aee39c443) to support our schemas the ga4gh/server is [~ 15 commits behind](https://github.com/ga4gh/server/pull/379) the schema project.  In order to proceed, we need to work with the community to help restart this effort.  I'd like to contribute to the server project with an implementation of /genotypephenotype/search backed with ohsu's cancer genomics database [ontology](http://nif-crawler.neuinfo.org/monarch/ttl/cgd.ttl).  I've commented on the PR [#379](https://github.com/ga4gh/server/pull/379), and will track progress.

**Comment by nlwashington** [comment]
Good idea to write your own implementation of the G2P schemas based on the cancer-drug data, but I think the data that is used in the reference server / compliance suite, should be of the highest quality, and a gold-standard set that is carefully crafted through this group and distributed through the git.  We should consider using something like yours for exemplar drug-phenotype-variant ternary associations.  We may need to extend the schemas first.

**Comment by bwalsh** [comment]
Emphatically agreed.  Ideally the test fixture we would provide for g2p
will coordinate and evolve with the [data](https://github.com/ga4gh/server/tree/develop/tests/data) used to test reads
and references

-Brian Walsh

On Wed, Jul 22, 2015 at 9:59 PM, Nicole Washington <notifications@github.com

**Comment by bwalsh** [comment]
This pair of PR will resolve this issue

ga4gh/server#607 ga4gh/schemas#432
